osc-server/src/main/java/org/osc/core/broker/rest/client/openstack/openstack4j/Openstack4JNeutron.java

```diff
@@ -258,11 +257,10 @@ private String getVmIdByPort(Port port) {
     }
 
     // Floating IP API
-    public List<String> getFloatingIpPools(String region, String projectId) throws Exception {
+    public List<String> getFloatingIpPools(String region) throws Exception {
         getOs().useRegion(region);
 
         Map<String, String> filter = Maps.newHashMap();
-        filter.put(QUERY_PARAM_TENANT_ID, projectId);
         filter.put(QUERY_PARAM_EXTERNAL_ROUTER, Boolean.TRUE.toString());
```

Comment:
As mentioned in first commet.. looks like route:external is a configuration map.